config/rbac/role.yaml

```diff
@@ -0,0 +1,28 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
+rules:
+- apiGroups:
+  - clustermanager.onprem.openshift.io
+  resources:
+  - joinedclusters
+  verbs:
+  - create
```

Comment:
Was this generated? If so, we should create a separate commit for generated code. If not, this seems like all the verbs where we can just use `'*'`.